src/java/org/apache/poi/hssf/usermodel/HSSFCell.java

```diff
@@ -42,14 +43,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 import org.apache.poi.ss.formula.eval.ErrorEval;
 import org.apache.poi.ss.formula.ptg.ExpPtg;
 import org.apache.poi.ss.formula.ptg.Ptg;
-import org.apache.poi.ss.usermodel.CellBase;
-import org.apache.poi.ss.usermodel.CellStyle;
-import org.apache.poi.ss.usermodel.CellType;
-import org.apache.poi.ss.usermodel.CellValue;
-import org.apache.poi.ss.usermodel.Comment;
-import org.apache.poi.ss.usermodel.FormulaError;
-import org.apache.poi.ss.usermodel.Hyperlink;
-import org.apache.poi.ss.usermodel.RichTextString;
+import org.apache.poi.ss.usermodel.*;
```

Comment:
We will adjust this when applying the diff, but in general please try to not change imports when not necessary, otherwise we would play import-ping-pong with changes by others.